modules/c++/math/include/math/Constants.h

```diff
@@ -23,6 +23,31 @@
 #ifndef __MATH_CONSTANTS_H__
 #define __MATH_CONSTANTS_H__
 
+#include <sys/sys_config.h>
+
+#ifdef __CODA_CPP11
+#include <math.h>
+namespace math
+{
+struct Constants
```

Comment:
I tried not defining them here, but because of the way that constexpr works, it requires you to give it a value as soon as you use it.  It basically attempts to evaluate a constexpr variable at compile time.  You also do still need static or else you can't access them without instantiating the class I believe.